packages/rich-text/src/component/index.native.js

```diff
@@ -264,7 +267,7 @@ export class RichText extends Component {
 	/*
 	 * Handles any case where the content of the AztecRN instance has changed
 	 */
-	onChange( event ) {
+	onChangeFromAztec( event ) {
```

Comment:
Updated method name to help distinguish from `this.props.onChange` which is used many times elsewhere in this class.